### PR TITLE
MAST/Kepler: Add a notebook on interactively inspecting Kepler data

### PR DESCRIPTION
**Pull request by barentsen**
This PR proposes to add one final Kepler-NumFocus notebook, titled *Interactively Inspecting Target Pixel Files and Light Curves*.

It demonstrates three Bokeh widgets which are offered by Lightkurve to explore Kepler data interactively:
* `interact()`
* `interact_sky()` 
* `interact_bls()`

This notebook was not ready in time to be part of #147, however it has been reviewed and copy-edited to the same standard of quality as the other notebooks.

I expect this will be the final PR.  /ping @mustaric

### CODE REVIEW

**Review by mustaric — APPROVED** [review]
I approve this notebook. The animated gifs are a good work around to make it instructive.

**Comment by pllim** [comment]
The GIFs are like 20 MB though.

File: notebooks/MAST/Kepler/kepler_interactively_inspecting_target_pixel_files_and_light_curves/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+lightkurve>=2.0b2
```
Comment (pllim) [inline notebooks/MAST/Kepler/kepler_interactively_inspecting_target_pixel_files_and_light_curves/requirements.txt:1]: Does lightkurve automatically pull in all the Bokeh stuff? I don't see `bokeh` defined here.

Comment (barentsen) [reply notebooks/MAST/Kepler/kepler_interactively_inspecting_target_pixel_files_and_light_curves/requirements.txt:1]: I confirm `bokeh` is a dependency of `lightkurve`.

**Comment by barentsen** [comment]
> The GIFs are like 20 MB though

I don't think we can make these much smaller unfortunately.  Do you expect it to be a major issue?

**Comment by pllim** [comment]
I'll wait a bit to see if @eteq has objection. If not, I'll merge tomorrow. 😸

**Review by pllim — APPROVED** [review]
:shipit:

**Comment by barentsen** [comment]
🙏

**Pull request merged**